knowledge-base/capturing-net-traffic.md

```diff
@@ -0,0 +1,131 @@
+
+- Execute your HTTPS requests from within the NET application.
+
+Fiddler captures the HTTPS sessions and displays them in the Live Traffic grid. 
+
+>tip Many Visual Studio users often rely on the built-in terminal during development. Fiddler can't work with the built-in terminal. However, you can configure your Visual Studio to attach the Fiddler's preconfigured terminal. This way, you can benefit from using the development perks (like debugging with breakpoints) while using the Fiddler proxy simultaneously. [Learn more about attaching running processes to the Visual Studio debugger here...](https://learn.microsoft.com/en-us/visualstudio/debugger/attach-to-running-processes-with-the-visual-studio-debugger?view=vs-2022).
+
+
+
+## Capturing NET Traffic through System Capturing
+
+A modern-day version of the NET framework respects the system proxy settings. To capture HTTPS traffic from the NET application, do the following:
+
+- Close your NET application.
+- Start the Fiddler Everywhere application.
+- Toggle ON the [system capturing mode]({%slug capture-traffic-get-started%}#system-capturing).
+- Start your NET application and make the HTTPS request.
+
+Fiddler captures the HTTPS sessions and displays them in the Live Traffic grid.
+
+
+
+## Manual Configuration
+
+Sometimes, you might need to manually configure your NET applications to explicitly respect the Fiddler Everywhere proxy. The below sections provide details on how to configure your [NET Framework](#configuring-proxy-in-net-application) or [NETCore applications](#configuring-proxy-in-netcore-application) in case the automatic traffic capturing from [preconfigured terminal](#capturing-net-traffic-through-preconfigured-terminal) or via the [system proxy](#capturing-net-traffic-through-system-capturing) is not applicable.
+
+
+### Configuring Proxy in NET Application
+
+To temporarily connect a NET application to the Fiddler Everywhere local proxy, you can use the [**WebRequest.DefaultWebProxy**](https://learn.microsoft.com/en-us/dotnet/api/system.net.webrequest.defaultwebproxy?view=net-8.0) class to set a proxy:
+
+```c#
+WebRequest.DefaultWebProxy = new WebProxy("127.0.0.1", 8866);
+```
+
+Alternatively, you can specify a proxy inside the **yourappname.exe.config** file. If the NET application is running in your current user account, add the following content inside the configuration section:
+
+```xml
+<configuration>
+    <system.net>
+        <defaultProxy>
+            <proxy bypassonlocal="false" usesystemdefault="true" />
+        </defaultProxy>
+    </system.net>
+</configuration>
+```
+
+If the NET application runs in a different user account (for example, a Windows service), edit the **machine.config** file:
```

Comment:
You should also add a note to revert the machine.config changes after FE is closed, because otherwise network traffic for the .NET service apps will stop.